extensions/default/src/DicomWebDataSource/utils/getWADORSImageId.js

```diff
@@ -6,7 +6,7 @@ function buildInstanceWadoRsUri(instance, config) {
 function buildInstanceFrameWadoRsUri(instance, config, frame) {
   const baseWadoRsUri = buildInstanceWadoRsUri(instance, config);
 
-  frame = frame != null || 1;
+  frame = frame || 1;
```

Comment:
This had been returning "true" for the frame number, which meant that frames couldn't be navigated to.

Reply:
So, multiframe was broken?
Does `frame` return `null` or `undefined`? This line returns 1 when frame is both `undefined` OR `null`.

Reply:
Yes, the multiframe was always requesting  .../frames/true  which seems to work for some DICOMweb implementations, but  not the static-wado one.